tools/its.py

```diff
@@ -283,20 +283,21 @@ def get_chrome_options(self, headless):
 
         # Chrome head is only required for the backup booking process.
         # User-Agent is required for headless, because otherwise the server lets us hang.
-        chrome_options.add_argument("user-agent=Mozilla/5.0")
+
+        # Echte useragents benutzen, zufaelligen waehlen
+
+        chrome_options.add_argument("user-agent="+self.useragent)
 
         chromebin_from_env = os.getenv("VACCIPY_CHROME_BIN")
         if chromebin_from_env:
             chrome_options.binary_location = os.getenv("VACCIPY_CHROME_BIN")
-        elif check_chromium():
-            chrome_options.binary_location = str(chromium_executable())
 
         chrome_options.headless = headless
 
         return chrome_options
 
     def get_chromedriver(self, headless):
-        return Chrome(self.get_chromedriver_path(), options=self.get_chrome_options(headless))
+        return uc.Chrome(options=self.get_chrome_options(headless))
```

Comment:
Welcher Chromedriver wird dafür verwendet? Sind unsere Chromedriver in `./tools/chromedriver` überflüssig?